specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2019-12-01-preview/definitions.json

```diff
@@ -0,0 +1,5157 @@
+            ]
+          },
+          "description": "Format in which the Api Details are exported to the Storage Blob with Sas Key valid for 5 minutes."
+        },
+        "value": {
+          "type": "object",
+          "description": "The object defining the schema of the exported Api Detail",
+          "properties": {
+            "link": {
+              "type": "string",
+              "description": "Link to the Storage Blob containing the result of the export operation. The Blob Uri is only valid for 5 minutes."
+            }
+          }
+        }
+      },
+      "description": "API Export result."
+    },
+    "ApiReleaseCollection": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApiReleaseContract"
+          },
+          "description": "Page values.",
+          "readOnly": true
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Next page link if any.",
+          "readOnly": true
+        }
+      },
+      "description": "Paged ApiRelease list representation."
+    },
+    "ApiReleaseContract": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ApiReleaseContractProperties",
+          "description": "ApiRelease entity contract properties."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./apimanagement.json#/definitions/Resource"
+        }
+      ],
+      "description": "ApiRelease details."
+    },
+    "ApiReleaseContractProperties": {
+      "properties": {
+        "apiId": {
+          "type": "string",
+          "description": "Identifier of the API the release belongs to."
+        },
+        "createdDateTime": {
+          "type": "string",
+          "readOnly": true,
+          "format": "date-time",
+          "description": "The time the API was released. The date conforms to the following format: yyyy-MM-ddTHH:mm:ssZ as specified by the ISO 8601 standard."
+        },
+        "updatedDateTime": {
+          "type": "string",
+          "readOnly": true,
+          "format": "date-time",
+          "description": "The time the API release was updated."
+        },
+        "notes": {
+          "type": "string",
+          "description": "Release Notes"
+        }
+      },
+      "description": "API Release details"
+    },
+    "ApiRevisionCollection": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApiRevisionContract"
+          },
+          "description": "Page values.",
+          "readOnly": true
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Next page link if any.",
+          "readOnly": true
+        }
+      },
+      "description": "Paged Api Revision list representation."
+    },
+    "ApiRevisionContract": {
+      "properties": {
+        "id": {
```

Comment:
It's on the top level, not inside "properties".

Reply:
@ryansbenson please see @vfedonkin comment. This is a contract associated with a proxy resource and we are just adding `id` to fix the ARM warnings generated as part of live traffic.

Reply:
@ryansbenson, @solankisamir , I think we will need to modify all entities to have our fields inside "properties", but it should be done in our next version, since we already behind the schedule for the version we trying to release.

Reply:
@ryansbenson a gentle ping on this!

Reply:
@ryansbenson we fixed this in the spec to unblock it.